src/main/scala/util/RunUtils.scala

```diff
@@ -68,38 +82,48 @@ object RunUtils {
       Interpret(IRProgram)
     }
 
+    Logger.info("[!] Removing external function calls")
+    // Remove external function references (e.g. @printf)
     val externalNames = externalFunctions.map(e => e.name)
     val externalRemover = ExternalRemover(externalNames)
+    // Removes BAP naming artefacts (e.g. # preceding variable names)
```

Comment:
That isn't what it does, it's renaming things that clash with reserved Boogie keywords